### ISSUE TITLE
Are top-right notifications received in realtime?

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/YouTube-operational-API#260]
I.e. without refreshing the webpage?

If so maybe there is a simple web-scraping to proceed to.

Would also benefit my OC3K [YouTube push notification](https://developers.google.com/youtube/v3/guides/push_notifications) callback.

**Comment by Benjamin-Loison** [comment]
No, I had opened:
- https://www.youtube.com without any interaction
- https://www.youtube.com with having shown and then hidden the notifications
- https://www.youtube.com with the notifications shown

None have been updated when thanks to YouTube push notification I have been notified that a new video has been published and I verified with another loaded for this purpose https://www.youtube.com that I noticed the notification.

In the 3 situations above if click on the notification bell then have the new entry.
Note this has been done with uBlock. Maybe a response with the new entry is received but not shown without interaction but I doubt so.
